docs/figma-to-code.md

```diff
@@ -1,254 +1,120 @@
+more.
+* **CSS Classes**: sets of styles that can be reused on multiple elements.  TeleportHQ will include
+some by default.  You can create, search, or edit classes as needed.  See their lesson on [CSS Classes]
+to learn more.
+* **Asset manager**: allows you to upload your own assets, access directly tens of thousands of images
+from the Unsplash, and choose icons from several popular libraries.
+
+#### Middle panel
+This is your **Canvas**, aka where all of the visual building ✨ magic ✨ happens.
+
+#### Right panel
+When an element is not selected, the right panel will display the **Design Language Panel**. This is
+where you can explore and define style tokens used throughout the project.  TeleportHQ includes some
+styles by default, e.g. "Primary" for colors. If your Figma export included color palettes, they will
+be listed under "Default" in the Color tab. Similarly, if your Figma export included text styles,
+they will be listed individually in the Text tab.
+
+When an element _is_ selected, the right panel will change to the **Style Inspector Panel**, where 
+you can add styles to the elements and/or edit the.
+
+> **Note** All tokens defined in the **Design Language Panel** will be included in the final code
+> export.  Feel free to add/modify/remove tokens as needed, so that the code export only has the
+> definitions you need.
+
+### Prepare the export
+
+Before exporting to code, we recommend reviewing the Figma-exported elements first, as the 
+Figma-to-teleportHQ export is not always perfect.
+
+For example, let's say you created a set of layers in Figma that is intended to be a button, that is,
+the layer group includes a shape, a color fill, and some text.  When this gets exported to TeleportHQ,
+it becomes as a group of layers that are an image (the color fill) and a text block.  This 
+subsequently gets exported as `<img>` and `<span>` in the code, which may not be what you intended.
+
+Instead, remove the image layer, then apply a CSS class to the parent group.  If you are using one
+of TeleportHQ's pre-defined CSS classes, you can edit it from the **CSS Classes** panel. You can
+alternatively use TeleportHQ's Button element from their **Elements** panel, and replace the "button"
+layers with it.
+
+If there is time or if you are not familiar with editing HTML & CSS, we recommend applying a layout
+to the Figma-exported project, e.g. applying the Row and Column elements.  Additionally, if 
+responsive design is important in the final product, we recommend using relative sizing and media
+queries.  See their lesson on [Responsive design] to learn more.
+
+Otherwise, you are more than welcome to add/edit the layout and responsiveness in the exported code.
+
+## Exporting the project
+
+1. In the top-right corner, select **Export** (the icon next to the Publish button).
+
+2. Select **Angular** or **HTML** as the library.  Note that exporting as **Angular** will download
```

Comment:
I guess it depends on whoever is doing it. I usually prefer HTML because that's easier for me to work with, but you may export as Angular since that's easier for you.

I'll make a comment that the choice is up to the user.